src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+  };
+
+  const visibleMovies = moviesFromServer.filter(
+    (movie) => movie.title.toLowerCase().includes(query.trim().toLowerCase())
+      || movie.description.toLowerCase().includes(query.trim().toLowerCase()),
```

Comment:
May have created separate variable for `query.trim().toLowerCase()` to not repeat yourself